pkg/handler/metadata/testdata/get_stacks/200.json

```diff
@@ -0,0 +1,25 @@
+{
+  "data": [
+    {
+      "id": "0ecf47c8-cca4-4c30-94bb-054b1124c44f",
+      "createdAt": "2022-11-11T18:38:46.266725Z",
+      "updatedAt": "2022-11-11T18:38:46.266725Z",
+      "Name": "Golang",
```

Comment:
start with a lowercase char

Reply:
I'm not sure because in previous task we all use lowercase for code an uppercase for name (chapter, position,...)

Reply:
thanks you, it's done